drivers/windows/R/resource.R

```diff
@@ -6,6 +6,7 @@ windows_cluster_info <- function(config, path_root) {
       max_cores = 32,
       max_ram = 512,
       queues = c("AllNodes", "Training"),
+      default_queue = "AllNodes",
```

Comment:
or, have the first element of queues be default?

Reply:
Hmm - I think I maybe prefer saying explicitly `default_queue` than `queues[1]` but not strongly so...